packages/ui/components/combobox/test/lion-combobox.test.js

```diff
@@ -2440,14 +2440,14 @@ describe('lion-combobox', () => {
       expect(_inputNode.value).to.equal('');
 
       el.setCheckedIndex(-1);
-      el.autocomplete = 'none';
+      el.autocomplete = 'list';
       el.setCheckedIndex(0);
       expect(_inputNode.value).to.equal('');
 
       el.setCheckedIndex(-1);
-      el.autocomplete = 'list';
+      el.autocomplete = 'none';
       el.setCheckedIndex(0);
-      expect(_inputNode.value).to.equal('');
+      expect(_inputNode.value).to.equal('Artichoke');
```

Comment:
This seems a bit contradictory tbh.

See:
- https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-none/
- https://www.w3.org/WAI/ARIA/apg/patterns/combobox/

```
The nature of possible values presented by a popup and the way they are presented is called the autocomplete behavior. Comboboxes can have one of four forms of autocomplete:
```